apps/web/src/presentation/components/launchpad/abc/ABCSettingsModelController.ts

```diff
@@ -32,6 +45,27 @@ export function useABCSettingsModelController(daoCreationRepository: DAOCreation
         init();
     }, []);
 
+    useEffect(() => {
+        async function checkBalance() {
+            const account = await getAccount();
+            const balance = await fetchBalance({
+                address: account.address as `0x${string}`,
+                token: daoCreationRepository.getDAOInfo().getABCConfig().collateralToken?.tokenAddress as `0x${string}`
+            });
+            const initialBalance = daoCreationRepository?.getDAOInfo()?.getABCConfig()?.getReserveInitialBalance() ?? 0
+            const parsedInitialBalance = parseEther(String(initialBalance));
+            if (initialBalance && balance.value >= parsedInitialBalance || initialBalance == 0) {
+                setEnoughBalance(true);
+            } else {
+                setEnoughBalance(false);
+            }   
+            console.log(enoughBalance);
+        }
+        checkBalance();
+    }, [
+        daoCreationRepository.getDAOInfo().getABCConfig().getReserveInitialBalance(), 
+        daoCreationRepository.getDAOInfo().getABCConfig().collateralToken?.tokenSymbol
```

Comment:
Can we make  a useEffect depend on a random variable that has not been defined with a useState? I don't think this will trigger.

Reply:
In my side is triggering without any problems, for both changing the token and the amount (although I see that the error messages are properly handled, but not the next button being disabled when it should)

Reply:
Already fixed - closing because outdated